lms/djangoapps/courseware/field_overrides.py

```diff
@@ -192,6 +209,17 @@ def get(self, block, name, default):  # pragma no cover
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def enabled_for(self, course):  # pragma no cover
+        """
+        Return True if this provider should be enabled for a given course
+
+        Return False otherwise
+
+        Concrete implementations are responsible for implementing this method
+        """
+        raise NotImplementedError
```

Comment:
Should this just return `False` by default, so that override providers have to be explicit in order to be enabled?